tests/platform_tests/test_intf_fec.py

```diff
@@ -18,6 +20,16 @@
     "100G", "200G", "400G", "800G", "1600G"
 ]
 
+FEC_CONFIG_DEFAULT_RETRIES = 3
+FEC_CONFIG_NON_COPPER_RETRIES = 10
+
+
+@pytest.fixture(scope="module", autouse=True)
+def disable_bgp(duthost):
+    duthost.shell("sudo config feature state bgp disabled")
```

Comment:
why we call it disable if we do disable and then enable?

Reply:
@liat-grozovik I think Mor want to disable the bgp feature during the test, and then enable it back after the test finished